scripts/falco-driver-loader

```diff
@@ -169,10 +169,15 @@ get_target_id() {
 		# 5.15.0-1009-aws -> ubuntu-aws
 		if [[ $KERNEL_RELEASE =~ -([a-zA-Z]+)(-.*)?$ ]];
 		then
-			TARGET_ID="ubuntu-${BASH_REMATCH[1]}${BASH_REMATCH[2]}"
+			TARGET_ID="ubuntu-${BASH_REMATCH[1]}"
```

Comment:
We will only use `-aws` here (without eventual `-5.19` suffix)

Reply:
I updated the above example!